lib/github/ldap.rb

```diff
@@ -68,6 +68,9 @@ class Ldap
     #
     def initialize(options = {})
       @uid = options[:uid] || "sAMAccountName"
+      @instrumentation_service = options[:instrumentation_service]
+      @admin_user = options[:admin_user]
+      @admin_password = options[:admin_password]
```

Comment:
The admin variables seem specific to AD. Is there another place that makes more sense to add these?

Is `@instrumentation_service` only kept to be passed into the initialzer of Net::LDAP?